docs/src/standard_results/field_outline.md

```diff
@@ -0,0 +1,88 @@
+# Field outline
+
+This exports the modelled field outline from within RMS.
+This field outline is a polygon representing the outline of the hydrocarbon-filled
+reservoir under initial conditions.
+
+The field outline is typically calculated as the intersection between the modelled
+top reservoir surface (in depth) and the fluid contact surface. Some assets may
+choose to further refine the outline by limiting it to areas with `net` reservoir
+properties. It is up to each asset to decide which definition best suits their field.
+
+The primary use case for the field outline is visualization. Unlike the deterministic
+outlines from NPD, it is derived directly from the reservoir model, providing greater
+precision, alignment with recent data, and adaptability to the unique structure and
+fluid contact of each realization.   
+
+:::{note} 
+It is only possible to export **one** field outline object per model workflow.
+For exporting outlines specific to different fluid types and zones, the
+`fluid_contact_outline` standard result should be used instead.
+:::
+
+:::{table} Current
+:widths: auto
+:align: left
+
+| Field | Value |
+| --- | --- |
+| Version | **{{ FieldOutlineSchema.VERSION }}** |
+| Output | `share/results/polygons/field_outline/field_outline.parquet` |
+:::
+
+## Requirements
+
+- RMS
+- a field outline polygon stored in the `General 2D data` folder within RMS.
```

Comment:
Is it possible to write to this with scripts? I.e. it is important that the outline is calculated during runtime.

Reply:
or, to rephrase, it is important that field outline exports are easy to do during runtime, where a new outline is calculated for each realization.

I think it might be good to include this also in the description. (I can add a comment there as well.)

Reply:
Yes it is possible to write to this location using scripts 🙂 either via xtgeo or the rms api. The plan is to add a functionality in e.g. `fmu-tools` to create these outlines and store to this particular location! When this is in place we will update the documentation here to reference it.

I will try to make it a bit clearer that these outlines should be calculated for every realization 👍

Reply:
Nice 👍 As for the clarifications, not sure it is needed. Your call.

Reply:
I think I'll just wait until we have some tooling in place to create them. That will probably help communicate the message that this outline is not intended to be deterministic 🙂